### PR TITLE
Update omni_channel_fraud_app.md

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@DT-Priyanka please review and merge the changes.

**Comment by sanjaypujare** [comment]
@DT-Priyanka  if you are happy can you merge?

File: docs/omni_channel_fraud_app.md [hunk 1]
```diff
@@ -1,32 +1,66 @@
...
+| AccountDataEnricher | This operator gets relevant user account security information from Account takeover prevention application using application backplane. |
+| Fraud Rules Executor | This operator is the Drools Operator. It applies the pre-defined rules to the incoming tuples and takes a suitable action depending on the outcome of the rules applied to a tuple. |
+| HDFS Output Operator | This output operator writes messages coming from the Rules Executor to the specified HDFS file path. Using this operator is optional in an FPA application. |
+| AOO Operator | This operator writes messages to a Kafka topic that are consumed by Online Analytics Service (OAS). |
 
 # Setting the Application
 
-Before you run the **Omni Channel Fraud Prevention** application, you must ensure to fulfill the prerequisites and to configure the Enrichment operators.
+Before you run the **Omni Channel Fraud Prevention** application, you must ensure to fulfill the prerequisites and to configure theenrichment.
```
Comment (DT-Priyanka) [inline docs/omni_channel_fraud_app.md]: the enrichment. Add space.

File: docs/omni_channel_fraud_app.md [hunk 2]
```diff
@@ -290,102 +431,82 @@ Update following properties as per your input load. The following properties mus
...
+3. Launch the application. Refer to [Launching Applications and Configurations](dtmanage.md)
+During the launch process, you can name the configuration and save it for future references. After you launch, you can track the details of the processed data in the application from the **Monitor** tab.
 
-![](images/applications/omni_channel_fraud_app/realtimefraudpreventionanalysis.png)
+# Generating Sample Input
 
-### Realtime FraudPrevention Operations
-The operations dashboard provides an operational view of the application so that the operations team can monitor the current performance of the application. It has metrics such as time series on latency of the rule executor, available free memory for rule executor, latency across the application, top 10 rules executed, number of process failures, and number of dropped transactions over time etc.
+For a test run, you may want to generate sample data. To generate sample data, run the **dt-cep-datagen-1.4.0.apa** application from the DataTorrent RTS interface. Specify kafka server details and topic name which must be the same as configured for Transaction Receiver.
```
Comment (DT-Priyanka) [inline docs/omni_channel_fraud_app.md]: To generate sample data, run the "OmniChannelFraudPreventionDataGenerator" applicatio from "dt-cep-omni-fraud-prevention-app-1.4.0.apa"

**Comment by lizygeogy** [comment]
@DT-Priyanka updated the comments. please review and merge.

**Pull request merged**